src/lib/components/Price.svelte

```diff
@@ -13,6 +13,6 @@
   });
 </script>
 
-<span class={twMerge("font-semibold text-success", clazz)}>
+<span class={twMerge("font-semibold", clazz)}>
```

Comment:
I do not agree with this design decision. I agree it doesn't look great in light mode, but I think we should find another greenish color for light mode.

Reply:
@fnurkla Can we get the nice green color back please? I think this design looks too "drowned out" and boring. 
<img width="478" alt="Screenshot 2024-05-10 at 22 06 28" src="https://github.com/Dsek-LTH/web/assets/29541261/d1d5d321-2158-426c-be3a-82e35fc367d5">